llvm/include/llvm/Transforms/Utils/BasicBlockUtils.h

```diff
@@ -611,6 +612,8 @@ LLVM_ABI void InvertBranch(BranchInst *PBI, IRBuilderBase &Builder);
 // br/brcond/unreachable/ret
 LLVM_ABI bool hasOnlySimpleTerminator(const Function &F);
 
+LLVM_ABI Printable printBBPtr(const BasicBlock *BB);
```

Comment:
Probably should take a reference, spell out BB, and not have Ptr in the name

Reply:
done, but can I leave it as a ptr? Because, otherwise, I'd have to check for null before the call and in that situation, I cannot use a ternary because depending on whether ptr is null or not, the ternary will evaluate to different types (Printable vs string literal), which isn't allowed. And without a ternary, everything becomes a little verbose